app/views/foreman_leapp/job_templates/leapp_upgrade.erb

```diff
@@ -18,7 +24,9 @@ template_inputs:
 - hosts: all
   tasks:
     - name: Run Leapp Upgrade
-      command: leapp upgrade
+% channel_opts = input('Channel') == 'none' ? '' : "--channel #{input('Channel')}" %>
```

Comment:
Syntax error, missing `<`. Plus it should be defined as an Ansible playbook variable

Reply:
Yeah, sorry: now is properly done.